### PR TITLE
Add legacy interface to mimick matlab code

### PR DESCRIPTION
**Pull request by stefsmeets**
This PR adds a legacy Python api call (`gemdat.legacy.analyse_md`) function to mimick the matlab code.

```python
from gemdat.legacy import analyse_md

data, sites, extras = analyse_md(
           '~/md-analysis-matlab-example/vasprun.xml',
           diff_elem='Li',
           supercell=(2,1,1),
           material='argyrodite',
           )
```

### Todo

- [x] Figure out why `trajectory_to_vasp_volume` crashes -> seems to be an issue on my side, needed to remove `vasprun.xml.cache`

### CODE REVIEW

**Comment by stefsmeets** [comment]
@v1kko I thought it would be fun to build a familiar interface into GEMDAT. For example, in a notebook environment this dumps all the plots into the output. 

What do you think?

File: src/legacy.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+    rdfs = calculate_rdfs(
+        data=data,
+        sites=sites,
+        diff_coords=extras.diff_coords,
+        n_steps=extras.n_steps,
+        equilibration_steps=extras.equilibration_steps,
+        max_dist=rdf_max_dist,
+        resolution=rdf_res,
+    )
```
Comment (v1kko) [inline src/legacy.py:98]: guard with if statement:
```suggestion
    if enable_rdf:
        rdfs = calculate_rdfs(
            data=data,
            sites=sites,
            diff_coords=extras.diff_coords,
            n_steps=extras.n_steps,
            equilibration_steps=extras.equilibration_steps,
            max_dist=rdf_max_dist,
            resolution=rdf_res,
        )
```

**Review by v1kko — APPROVED** [review]
Looks good, works for me in a notebook, nice that the progress bar also works.

I have one nitpick, currently the function has to be invoked with 
```
from gemdat import legacy
legacy.analyse_md("vasprun.xml",diff_elem="Li",material="argyrodite")
```
but I'd much rather do
```
from gemdat.legacy import analyze_md (OR) from gemdat import analyze_md
analyse_md("vasprun.xml",diff_elem="Li",material="argyrodite")
```

And since calculating RDFs can take a long time, and you lose all output if it fails, should we include a toggle for it (enable_rdf)? See suggestions, but I think its better to just ignore those and make a single commit.

File: src/legacy.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    diffusion_dimensions: int = 3,
+    z_ion: int = 1,
+    nr_parts: int = 10,
+    dist_collective: float = 4.5,
+    density_resolution: float = 0.2,
+    jump_res: float = 0.1,
+    rdf_res: float = 0.1,
+    rdf_max_dist: int = 10,
+    start_end: tuple[int, int] = (5000, 7500),
+    nr_steps_frame: int = 5,
```
[the remark this answers is not in the document]
Comment (stefsmeets) [reply src/legacy.py:26]: Good point, this is also in the original matlab code :+1:

**Pull request merged**